api/ws/rpc/connect.js

```diff
@@ -45,20 +45,22 @@ const connectSteps = {
 	addConnectionOptions: peer => {
 		const systemHeaders = System.getHeaders();
 		const queryParams = {};
-
-		if (systemHeaders.version != null) {
+		if (systemHeaders.version !== null && systemHeaders.version !== undefined) {
```

Comment:
why not just `if(systemHeaders.version)` ?
reference: https://runkit.com/1manugowda/5bf7fc40fbd88200127181b8

Reply:
https://github.com/LiskHQ/lisk/pull/2562#discussion_r235380896

Reply:
Your suggestion @ManuGowda returns false for the following: `"", false, 0, undefined, null`, whereas the code below only for `undefined, null`.

Reply:
I want to keep the behaviour while making the statement easier to understand. Please, raise an issue if you think we need to rewrite/clean that function.